store/tikv/coprocessor.go

```diff
@@ -219,6 +213,50 @@ const rangesPerTask = 25000
 
 func buildCopTasks(bo *Backoffer, cache *RegionCache, ranges *copRanges, req *kv.Request) ([]*copTask, error) {
 	start := time.Now()
+	rangesLen := ranges.len()
```

Comment:
This line can be removed if we call `ranges.len()` when logging.

Reply:
Done